### DISCUSSION TITLE
Sobre fones de ouvido para programar

### DISCUSSION THREAD

**Discussion by danjesus** [frontendbr/forum#377] [Discussão]
Pessoal, estou querendo comprar um fone de ouvido para programar e queria saber quais vocês usam/indicam?

**Comment by robsonbittencourt** [comment]
Eu uso este, e recomendo:
http://www.sony.com.br/electronics/headphones-sobre-as-orelhas/mdr-10rnc

Na minha opinião o Noise Canceling é um grande diferencial, pois trabalho em um lugar barulhento.

**Comment by gabrieldarezzo** [comment]
Caso esteja buscando um modelo custo beneficio por preço baixo os modelos da (MDR-ZX-*) Sony acho que tem uma qualidade bem bacana.

Se não estiver disposto a gastar horrores e quiser uma qualidade recomendo xD

Eu não consigo utilizar o fone padrão pelo modelo da minha orelha... eles ficam caindo e isso me deixa extremamente irritado, então vai de pessoa pra pessoa.

Sobre o isolamento que o @robsonbittencourt comentou, tmb concordo.
Atualmente trabalho na mesma sala que a galera do suporte, então o dia todo eles atendendo telefones, ai nesse caso a musica acaba até ajudando na concentração.

**Comment by danjesus** [comment]
@gabrieldarezzo Justamente o isolamento é o que mais me interessa, valeu pela dica vou dar uma olhada.

**Comment by RicardoJunior** [comment]
Cara, eu tive um Marshall, um Bose, e agora um Beats Pro.

Meu problema com o Marshall foi durabilidade, em poucos meses de uso o isolamento já tava bem merda, por assim dizer e em compensação todo seu acabamento ainda tava bonitinho. Depois comprei o Bose, que talvez tenha sido um problema pessoal comigo e ele não durou quase nada comigo, foram tipo 3 meses, o isolamento já tava ruim, o couro do acabamento dele já tava todo rachado, e eu usava só pra trabalhar. Me restaram de opções Beats e JBL. Eu acabei comprando um beats, pq na época não tinha achado um JBL OverEar com uma entrega em prazo pequeno e o Beats Pro o submarino entregava em 1 dia pq era um produto da categoria CoolStuff. O meu Beats Pro durou 2 anos e meio, ele ta começando a rachar agora, mas ele me impressionou bastante, eu achava que era "mais um fone hipster". Agora eu vou comprar um JBL e espero que seja bom.

**Comment by kelciocajueiro** [comment]
Eu tinha um AKG K67 Tiesto que durou exatos 1 ano. Ele é excelente, o problema é que as hastes não são muito flexíveis então quebram rápido. Hoje utilizo um simples intra-auricular da Sony (MDREX15LP/BCE) e quando estou ouvindo música não dá pra ouvir nada do ambiente externo.

**Comment by paulinhoerry** [comment]
Sennheiser HD 598, tenho um e posso dizer, melhor fone que já tive.

**Comment by nighto** [comment]
Também recomendo os fones da Sony. Adquiri em agosto o [MDR-ZX770BNB](http://www.sony.com/electronics/headband-headphones/mdr-zx770bn). Ele é over-ear e é bem grande, cobrindo completamente a orelha, sem ficar sobre ela - o que é essencial para quem usa óculos, pois fones on-ear tradicionalmente apertam o lóbulo da orelha na haste do óculos, o que incomoda muito depois de uma ou duas horas.

É bluetooth, mas também tem um plug P2 fêmea, o que permite usar com fio no computador ou no celular caso a bateria acabe ou você prefira não parear via bluetooth. Outra possibilidade: parear no celular o bluetooth e usar no P2 no computador (embora não dê para fazer as duas coisas ao mesmo tempo, ao conectar o cabo ele corta a conexão BT).

Esse modelo em específico tem noise-cancelling, não tão forte como um Bose QuietConfort 35, mas consideravelmente mais barato. Paguei €99 em Berlim (MediaMarkt, loja física), enquanto o QC35 saía por €349.

Em resumo: se tiver bala na agulha pra pagar quase mil e quinhentas pratas e não se incomodar em usar pilhas palito, pega o Bose QC35, se não pega esse Sony (que tem bateria interna e carrega na Micro USB).

**Comment by barbier** [comment]
Estou com um LX-3000 há 5 anos e recomendo!

**Comment by gss-patricia** [comment]
Eu tbm recomendo o LX-3000, estou com o meu faz 4 anos em perfeito estado.

**Comment by felipefialho** [comment]
Rolou essa [thread sensacional com milhares de indicações incríveis](https://twitter.com/LFeh/status/1060621073841119234), compartilhando aqui para escalar 😄
